lib/wallet/balance-tracker-contract.js

```diff
@@ -1,13 +1,15 @@
 'use strict';
 
 const ethers = require('ethers');
-const balanceTrackerDeployment = require('./abis/BalanceTracker');
 
 class BalanceTrackerContract extends ethers.Contract {
     constructor(walletOrProvider) {
         const provider = walletOrProvider.provider || walletOrProvider;
+        const networkName = provider.network.name;
+        const balanceTrackerDeployment = require(`./abis/${networkName}/BalanceTracker`);
```

Comment:
Is there a risk of case sensitivity issues here? What if the returned network name is capitalized?

Reply:
From my understanding network names are supposed to be case sensitive. If the file can not be found (e.g.: "rinkeby" as network) this will lead to a hard crash exit by NodeJS, which I think is the desired outcome anyway.